File: languages/java/lib/src/main/java/bloock/sdk/java/entity/RecordReceipt.java
package bloock.sdk.java.entity;

import bloock.sdk.java.bridge.proto.RecordOuterClass;

public class RecordReceipt {
    long anchor;
    String client;
    String record;
    String status;

    RecordReceipt(long anchor, String client, String record, String status) {
        this.anchor = anchor;
        this.client = client;
        this.record = record;
        this.status = status;
    }

    public static RecordReceipt fromProto(RecordOuterClass.RecordReceipt receipt) {
        return new RecordReceipt(
                receipt.getAnchor(),
                receipt.getClient(),
                receipt.getRecord(),
                receipt.getStatus());
    }

    RecordOuterClass.RecordReceipt toProto() {
        return RecordOuterClass.RecordReceipt
                .newBuilder()
                .setAnchor(anchor)
                .setClient(client)
                .setRecord(record)
                .setStatus(status)
                .build();
    }

	public long getAnchor() {
		return anchor;
	}

	public String getClient() {
		return client;
	}

	public String getRecord() {
		return record;
	}

	public String getStatus() {
		return status;
	}
}

File: languages/php/Bloock/Entity/RecordReceipt.php
<?php

namespace Bloock\Entity;

class RecordReceipt
{
    private int $anchor;
    private string $client;
    private string $record;
    private string $status;

    public function __construct(int $anchor, string $client, string $record, string $status) {
        $this->anchor = $anchor;
        $this->client = $client;
        $this->record = $record;
        $this->status = $status;
    }

    public static function fromProto(\Bloock\RecordReceipt $receipt): RecordReceipt {
        return new RecordReceipt($receipt->getAnchor(), $receipt->getClient(), $receipt->getRecord(), $receipt->getStatus());
    }

    public function toProto(): \Bloock\RecordReceipt {
        $p = new \Bloock\RecordReceipt();
        $p->setAnchor($this->anchor);
        $p->setClient($this->client);
        $p->setRecord($this->record);
        $p->setStatus($this->status);
        return $p;
    }

    /**
     * @return int
     */
    public function getAnchor(): int
    {
        return $this->anchor;
    }

    /**
     * @return string
     */
    public function getClient(): string
    {
        return $this->client;
    }

    /**
     * @return string
     */
    public function getRecord(): string
    {
        return $this->record;
    }

    /**
     * @return string
     */
    public function getStatus(): string
    {
        return $this->status;
    }
}